data/swtpm/ssh_port_chk_script

```diff
@@ -3,10 +3,11 @@ set -euo pipefail
 
 ip_addr=""
 for (( i = 0; i < 200; i++ )); do
-    ip_addr=`ip n | awk ' /192\\.168/ {print $1}'`
+    # The default network of libvirt is 192.168.122.*
+    ip_addr=`ip n | awk ' /192\\.168\\.122/ {print $1}'`
     if [[ -n "$ip_addr" ]] && nc -zvw2 $ip_addr 22 2>/dev/null; then
```

Comment:
I think 'nc -zv $ip_addr 22' is enough, what's your opinion.

Reply:
I think we need to ask @rfan1 for his opinions.